src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -201,6 +217,19 @@ private void handleJobElement(ImmutableList.Builder<KeyValueLabelMetadataEntity>
         jobHandled = true;
     }
 
+    private void handleDescriptionElement(ImmutableList.Builder<KeyValueLabelMetadataEntity> keyValueMapBuilder,
+            XMLStreamReader xmlStreamReader) {
+
+        String descriptionContent = new String();
+        try {
+            descriptionContent = xmlStreamReader.getElementText(); //returns the text content of CDATA for description tag in our case
+        } catch (XMLStreamException e) {
+            log.error("Unable to parse the workflow description", e);
```

Comment:
Are you sure you dont want to throw this exception?

Reply:
If the `description` tag is not possible to parse, I don't want to block execution or stop anything with throwing an exception. Because this filed is not so important and can be empty in case of error.
The alternative is to throw an exception, stop the creation of object with error UnprocessableEntityException. What do you think?

Reply:
if there is no way to check existence of this element before accessing it, then fine.

Reply:
actually inside this method we enter only when the description is exist. So this implementation should throw an exception